superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -109,11 +109,6 @@ export function setExploreControls(formData: QueryFormData) {
   return { type: SET_EXPLORE_CONTROLS, formData };
 }
 
-export const REMOVE_CONTROL_PANEL_ALERT = 'REMOVE_CONTROL_PANEL_ALERT';
-export function removeControlPanelAlert() {
-  return { type: REMOVE_CONTROL_PANEL_ALERT };
-}
```

Comment:
Cleaning up dead code.

<img width="530" alt="alert-deprecated" src="https://user-images.githubusercontent.com/335541/108408702-391dde00-71da-11eb-811e-ffdaf3be596a.png">

This is the alert this action is trying to clear, which we are not using anymore.